grafana-plugin/src/containers/TeamName/TeamName.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { Badge, Tooltip } from '@grafana/ui';
+import { observer } from 'mobx-react';
+
+import Avatar from 'components/Avatar/Avatar';
+import Text from 'components/Text/Text';
+import { GrafanaTeam } from 'models/grafana_team/grafana_team.types';
+
+interface TeamNameProps {
+  team: GrafanaTeam;
+}
+
+const TeamName = observer((props: TeamNameProps) => {
+  const { team } = props;
+  return (
+    <>
+      {team.id === 'null' ? (
+        <Badge text={team.name} color={'blue'} tooltip={'Resource is not assigned to any team (ex General team)'} />
+      ) : (
+        <Text type="secondary">
+          <Avatar size="small" src={team.avatar_url} />{' '}
+          <Tooltip placement="top" content={'Resource is assigned to ' + team.name}>
+            <span>{team.name}</span>
+          </Tooltip>{' '}
+        </Text>
+      )}
+    </>
+  );
```

Comment:
```suggestion
  if (team.id === 'null') {
    return <Badge text={team.name} color={'blue'} tooltip={'Resource is not assigned to any team (ex General team)'} />;
  }
  return (
    <Text type="secondary">
      <Avatar size="small" src={team.avatar_url} />{' '}
      <Tooltip placement="top" content={'Resource is assigned to ' + team.name}>
        <span>{team.name}</span>
      </Tooltip>{' '}
    </Text>
  );
```

Few other questions:
- should `team.id` be the literal string of `'null'` or actual `null`?
- we are getting rid of the idea of the "General" team, right? If so, consider rewording the tooltip message.

Reply:
I've added it to remove confusion after the update. Otherwise we could silently rename General to "no team"
<img width="964" alt="Screenshot 2023-03-20 at 12 55 02" src="https://user-images.githubusercontent.com/2262529/226251339-f2c4a3ac-2cee-4ab9-8b2a-9dfda3cc9964.png">